refinery/annotation_server/management/commands/add_genome_annotation.py

```diff
@@ -16,7 +16,6 @@
 # Helper command to deal with additional annotation tracks not available from
 # UCSC genome browser
 #
-# TODO: Should we import annotation files into file_server?
 # how to cleanup directories used to store and download files
```

Comment:
This line is probably a TODO in its own right?

Reply:
I was confused about this line. Should I prefix it with a TODO?